tests/conftest.py

```diff
@@ -59,61 +63,134 @@ def cmd_line_args(request):
     return args
 
 
+MYDIR = os.path.dirname(os.path.realpath(__file__))
+
 FIXTURE_FILES_DIR = "/tmp/"
 FIXTURE_FILES_PREFIX = "hopr-smoke-test"
 
+NODE_NAME_PREFIX = "hopr-smoke-test-node"
+
+ANVIL_CFG_FILE = f"{FIXTURE_FILES_DIR}{FIXTURE_FILES_PREFIX}-anvil.cfg"
+ANVIL_LOG_FILE = f"{FIXTURE_FILES_DIR}{FIXTURE_FILES_PREFIX}-anvil.log"
+ANVIL_NETWORK1 = "anvil-localhost"
+ANVIL_NETWORK2 = "anvil-localhost2"
+
+PROTOCOL_CONFIG_FILE = f"{MYDIR}/../packages/hoprd/crates/hopr-lib/data/protocol-config.json"
+DEPLOYMENTS_SUMMARY_FILE = f"{MYDIR}/../packages/ethereum/contracts/contracts-addresses.json"
+
+PREGENERATED_IDENTITIES_DIR = f"{MYDIR}/identities"
 
 DEFAULT_API_TOKEN = "e2e-API-token^^"
 PASSWORD = "e2e-test"
 NODES = {
     "1": {
-        "p2p_port": 19091,
-        "api_port": 13301,
-        "peer_id": "12D3KooWKSzQgdszZzipRVGSRwBcC3etYwjSmqqTqcySn97EGWTm",
-        "address": "0x7d1e530e9c82c21b75644a2c23402aa858ae4a69",
+        "api_port": 19091,
+        "p2p_port": 13301,
+        "api_token": DEFAULT_API_TOKEN,
+        "dir": f"{FIXTURE_FILES_DIR}{NODE_NAME_PREFIX}_1",
+        "host_addr": "localhost",
     },
     "2": {
-        "p2p_port": 19092,
-        "api_port": 13302,
-        "peer_id": "12D3KooWLWoHJjaS1z9cXn19DE9gPrSbYHkf7CHMbUtLUqbZKDby",
-        "address": "0x1b482420afa04aec1ef0e4a00c18451e84466c75",
+        "api_port": 19092,
+        "p2p_port": 13302,
+        "dir": f"{FIXTURE_FILES_DIR}{NODE_NAME_PREFIX}_2",
+        "host_addr": "127.0.0.1",
     },
     "3": {
-        "p2p_port": 19093,
-        "api_port": 13303,
-        "peer_id": "12D3KooWJ4E4q6wr8nzXyRKnAofQSeoGoRFRRtQJK3jCpLtLNVZj",
-        "address": "0x05b17e37fd43c18741877fca80846ad8c84aa750",
+        "api_port": 19093,
+        "p2p_port": 13303,
+        "api_token": DEFAULT_API_TOKEN,
+        "dir": f"{FIXTURE_FILES_DIR}{NODE_NAME_PREFIX}_3",
+        "host_addr": "localhost",
     },
     "4": {
-        "p2p_port": 19094,
-        "api_port": 13304,
-        "peer_id": "12D3KooWA494BRhXs2DpMm5e2DWkPZcot3WpYwB4KBj2udP9xvPC",
-        "address": "0xcc70a22331998454160472f097acb43ca9b1e646",
+        "api_port": 19094,
```

Comment:
Was switching p2p port and API port prefixes intentional? `1330x` to `1909x` and vice versa.

Reply:
Nice catch.